aws/sdk/integration-tests/webassembly/src/default_config.rs

```diff
@@ -3,25 +3,21 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-use aws_config::retry::RetryConfig;
 use aws_credential_types::Credentials;
-use aws_smithy_types::timeout::TimeoutConfig;
+use aws_smithy_types::{retry::RetryConfig, timeout::TimeoutConfig};
 use aws_types::region::Region;
-use std::future::Future;
-
-use crate::adapter::Adapter;
+use std::{env, future::Future};
 
 pub(crate) fn get_default_config() -> impl Future<Output = aws_config::SdkConfig> {
     aws_config::from_env()
         .region(Region::from_static("us-west-2"))
         .credentials_provider(Credentials::from_keys(
-            "access_key",
-            "secret_key",
-            Some("session_token".to_string()),
+            env::var("AWS_ACCESS_KEY_ID").expect("AWS_ACCESS_KEY_ID"),
+            env::var("AWS_SECRET_ACCESS_KEY").expect("AWS_SECRET_ACCESS_KEY"),
+            env::var("AWS_SESSION_TOKEN").ok(),
```

Comment:
this shouldn't be needed since this behaves just like the default chain that will pull creds from the environment

Reply:
Yes, I will try to put back the default provider chain.